browserslist-config-terra.js

```diff
@@ -1 +0,0 @@
-module.exports = require('./packages/browserslist-config-terra');
```

Comment:
did we want to drop this?

Reply:
Yes, unfortunately this wasn't working as intended. A little more explanation on this comment:

https://github.com/cerner/terra-toolkit/pull/406#discussion_r420970349